ripple/src/main/webapp/maps.js

```diff
@@ -6,39 +6,95 @@ script.defer = true;
 script.async = true;
 document.head.appendChild(script); 
 
-/* Render a map */
+/* Render a world map centered at the US. Read user address and business address
+ from local Storage and Firestore, respectively. Read business name and street from 
+ Firestore and add to DOM */
 window.initMap = () => {
-  const directionsService = new google.maps.DirectionsService();
-  const directionsRenderer = new google.maps.DirectionsRenderer();
-  const map = new google.maps.Map(document.getElementById("map"), {
-    zoom: 8,
-    // TODO: Import user's current location from local storage
-    center: { lat: 34.05, lng: -118.24 },
-    disableDefaultUI: true,
-    zoomControl: true,
-  });
-  directionsRenderer.setMap(map);
-  calculateAndDisplayRoute(directionsService, directionsRenderer);
+
+  // TEMPORARY LOCAL STORAGE 
+  localStorage.setItem("businessId", "uxq1XtM4zIHqxmdnjnf0");
+  localStorage.setItem("inputAddress", "398 S Willaman Dr, Los Angeles, CA 90048");
+  localStorage.setItem("walkingTime", 15); 
+  localStorage.setItem("walkingDist", 0.7);
+
+  // Read local storage values
+  businessId = localStorage.getItem("businessId");
+  inputAddress = localStorage.getItem("inputAddress");
+  walkingTime = localStorage.getItem("walkingTime");
+  walkingDist = localStorage.getItem("walkingDist");
+
+  // Add walking time to DOM
+  addTextToDom(walkingTime, "bd-walking-dist");
+
+  // Access 'businesses' collection to read the business name and address 
+  var lambda = (doc) => {
+        if (doc.exists) {
+          // Read business name and add to DOM 
+          businessName = doc.data().businessName[1];
+          addTextToDom(businessName, "bd-header");
+
+          // Read business address and add street to DOM. 
+          addressArray = doc.data().address;
+          addTextToDom(addressArray[0], "bd-address");
+          // Join each array element of the addressArray into a string. Set as the map destination.
+          destination = addressArray.join(" ");
+          console.log("destination address:" + destination);
+
+          // Specify map display and initialize directions service
+          const directionsService = new google.maps.DirectionsService();
+          const directionsRenderer = new google.maps.DirectionsRenderer();
+          var centerOfUnitedStates = new google.maps.LatLng(39.8283, -98.5795);
+          const map = new google.maps.Map(document.getElementById("map"), {
+            zoom: 3,
+            center: centerOfUnitedStates,
+            // Only display zoom settings from UI controls
+            disableDefaultUI: true,
+            zoomControl: true,
+          });
+          directionsRenderer.setMap(map);
+          calculateAndDisplayRoute(map, directionsService, directionsRenderer, destination);
+
+        } else {
+          console.log("Error: business document does not exist");
+        }
+      }
+  getDocByDocId("businesses", businessId, lambda);
```

Comment:
maybe give a more descriptive name for lambda?

Reply:
Fixed.